scripts.py

```diff
@@ -1381,22 +1406,26 @@ def load_collection(self, collection_name, data_source_name):
             )
 
             try:
-                mirror_integration = MirrorUploader.sitewide_integration(
-                    self._db
+                mirror_integration = MirrorUploader.integration_by_name(
+                    self._db, storage_name
                 )
             except CannotLoadConfiguration, e:
-                # There is no sitewide mirror configuration, or else
-                # there is more than one. Either way, we can't
-                # associate a mirror integration with the new collection.
+                # We can't associate any mirror integration with the new collection.
```

Comment:
Yeah, looking at this I don't think it makes sense anymore. The two calls to `MirrorUploader.for_collection` below are more what we need.